apps/perf-test/src/scenarioNames.js

```diff
@@ -0,0 +1,14 @@
+const scenarioNames = {
+  BaseButtonNew: 'BaseButton (experiments)',
+  DefaultButtonNew: 'DefaultButton (experiments)',
+  DetailsRowNoStyles: 'DetailsRow without styles',
+  DocumentCardTitle: 'DocumentCardTitle with truncation',
+  MenuButtonNew: 'MenuButton (experiments)',
+  PrimaryButtonNew: 'PrimaryButton (experiments)',
+  SplitButtonNew: 'SplitButton (experiments)',
+  StackWithIntrinsicChildren: 'Stack with Intrinsic children',
+  StackWithTextChildren: 'Stack with Text children',
+  ToggleNew: 'Toggle (experiments)'
```

Comment:
Since we are already adding the `Toggle` in `experiments`, should we also add the one that is already in `office-ui-fabric-react`?

Reply:
It's already there. This is just an optional naming structure. `Toggle` is left to its own name of `Toggle` so doesn't appear here.